backend/src/schema/resolvers/users/blockedUsers.spec.js

```diff
@@ -20,7 +20,7 @@ beforeEach(() => {
       return {
         user: authenticatedUser,
         driver,
-        neode: instance,
+        neode: neode,
```

Comment:
```suggestion
        neode,
```
same as :point_up_2: